packages/v4/patternfly-docs/content/design-guidelines/components/form/forms.md

```diff
@@ -200,7 +204,7 @@ Examples:
 
 For more information about writing effective placeholder text, see the [content](#content) section.
```

Comment:
Since the content section is referred to multiple times, I'd do a control+F on all the times the content section is mentioned and make sure the link is updated to match the new content considerations header